fxa-auth-db-server/docs/API.md

```diff
@@ -1525,6 +1640,51 @@ Content-Length: 2
     * Body : {"code":"InternalError","message":"...<message related to the error>..."}
 * ``
 
+## verifyToken : `POST /token/<tokenVerificationId>/verify`
+
+This method verifies sessionTokens and keyFetchTokens.
+Note that it takes the tokenVerificationId
+specified when creating the token,
+NOT the tokenId.
+
+### Example
+
+```
+curl \
+    -v \
+    -X POST \
+    -H "Content-Type: application/json" \
+    -d '{}' \
+    http://localhost:8000/token/266fd690895c8b0086bb2c83e4b3b41c128746125f28b5429938765279673d62/verify
+```
+
+### Request
+
+* Method : POST
+* Path : `/token/<tokenVerificationId>/verify`
+    * tokenVerificationId : hex256
+* Params:
+    * tries : int
+
+### Response
+
+```
+HTTP/1.1 200 OK
+Content-Type: application/json
+Content-Length: 2
+
+{}
+```
+
+* Status Code : 200 OK
+    * Content-Type : 'application/json'
+    * Body : {}
+* Status Code : 500 Internal Server Error
```

Comment:
> We have link expired errors for signup and password reset tokens.

Ha, I stand corrected. :smile: 

In that case, assuming we want to do the same thing here, I'll make it 404.